### DISCUSSION TITLE
irdb.tk down

### DISCUSSION THREAD

**Discussion by benjaminmetzler** [probonopd/irdb#65] [General]
Will it come back up?  Also are there any cli tools to convert between irdb/csv and lirc.conf formats?

Thanks,
Ben

**Comment by bengtmartensson** [comment]
> Will it come back up?

Personally, I consider it very unlikely that the web site irdb.tk will come back. (But this is just my view...) This GitHub repository will probably remain.

> Also are there any cli tools to convert between irdb/csv and lirc.conf formats?

First check out [IrScrutinizer](https://github.com/bengtmartensson/IrScrutinizer), which can import and export all the requested formats. (See #56.) But it does not do very much command line. Check out [RemoteLocator](https://github.com/bengtmartensson/RemoteLocator). Exactly what do you want to achieve?

**Reply by benjaminmetzler** [reply]
That's too bad.  It looked like it was useful.  Is there a reason it went down?

Basically I want to see about integrating irdb into the [my_remote project](https://github.com/benjaminmetzler/my_remote) for searching and adding new IR components, similar to the wizards a lot of tools have.

**Comment by probonopd** [comment]
The .tk domain was provided since 2013 by Freenom and was a pain to keep running, because you have to re-register annually during a short time window and if you miss it, you lose the domain. Also, this year they won't even let me  do this because they have set the status of the domain to "FRAUD", whatever they mean by that (no fraud going on whatsoever). Possibly triggered by the web hosting company having upgraded their server which broke my web service.

Since I am doing this purely as a hobby, I do not want to have to have to worry about hosting companies, bills, updates, and keeping the server running by constantly nurturing it. So I am looking for a solution that can be hosted as a static website (e.g., generated by GitHub Actions) on GitHub Pages or something like it.

**Reply by GGORG0** [reply]
Can anyone publish its source code or something? I really want to use the code explorer and there is no other solution. If not the source code, then idk, maybe something else. But I just want to self host it for a minute and sue one thing from it

**Reply by probonopd** [reply]
Unfortunately I don't have it anymore. Did not continue paying the bills.

**Reply by bengtmartensson** [reply]
What do you want, really? The data is available here; if you want to transform it then [IrpTransmogrifier](https://github.com/bengtmartensson/IrpTransmogrifier) is the tool (GUI access to much of the functionality through IrScrutinizer), the source code for the web site as such: well, Probono says it is gone, too bad, but there is no real need for it. And please try the RemoteLocator in IrScrutinizer.

**Reply by GGORG0** [reply]
> What do you want, really? 

Well, the DIYRemote app has integration with the website using the widgets tab. If i wanted to make a universal remote myself, it would take a lot if time to export all the codes and make some html for it. Maybe I could make a script or something, but still
